cns/NetworkContainerContract.go

```diff
@@ -35,6 +35,7 @@ const (
 	Basic                  = "Basic"
 	JobObject              = "JobObject"
 	COW                    = "COW" // Container on Windows
+	AKS                    = "AKS"
```

Comment:
Discussed with Tamilmani, lets remove it, we will be using ContainerType = "Docker"